inputs/genome_contaminants.csv

```diff
@@ -0,0 +1,21 @@
+﻿ident,superkingdom,phylum,class,order,family,genus,species

```

Comment:
Is the current setup so somebody can just reproduce how you created these databases, or if they wanted to add a genome contaminant to what you have can they either add a line to this CSV or provide a completely different CSV and it still works?

Reply:
they'd have to add a line to this CSV